src/applications/vaos/components/calendar/CalendarRow.jsx

```diff
@@ -85,49 +85,54 @@ export default class CalendarRow extends Component {
         );
 
         return (
-          <div className={cssClasses}>
-            {optionsError && (
-              <span
-                className="usa-input-error-message vads-u-margin-bottom--2 vads-u-padding-top--0 vads-u-width--full"
-                role="alert"
-              >
-                <span className="sr-only">Error</span> {optionsError}
-              </span>
-            )}
-            {selectedDateOptions.map((o, index) => {
-              const dateObj = {
-                date: currentlySelectedDate,
-                [fieldName]: o.value,
-              };
-              const checked = isDateOptionPairInSelectedArray(
-                dateObj,
-                selectedDates,
-                fieldName,
-              );
+          <fieldset>
+            <legend className="vads-u-visibility--screen-reader">
+              {additionalOptions.legend}
```

Comment:
As long as we always have text here, I'm good to pass this PR. Without text, assistive tech users are left to figure out what AM/PM and the times represent.